website/content/en/preview/upgrade-guide.md

```diff
@@ -103,7 +103,7 @@ Snapshot releases are tagged with the git commit hash prefixed by the Karpenter
 ## Released Upgrade Notes
 
 ### Upgrading to v0.29.0+
-* Karpenter has changed the default metrics service port from 8080 to 8000. In `v0.28.0`, the Karpenter pod port was changed to 8000, but referenced the service by name, allowing users to scrape the service at port 8080 for metrics. `v0.29.0` aligns the two ports so that default service and default pod metrics ports are the same.
+* Karpenter has changed the default metrics service port from 8080 to 8000. In `v0.28.0`, the Karpenter pod port was changed to 8000, but referenced the service by name, allowing users to scrape the service at port 8080 for metrics. `v0.29.0` aligns the two ports so that service and pod metrics ports are the same.These ports are both set by `controller.metrics.port` and `webhook.port`, so if you have previously set these to non-default values, you may need to update your Prometheus scraper to match these new values.
```

Comment:
```suggestion
* Karpenter has changed the default metrics service port from 8080 to 8000. In `v0.28.0`, the Karpenter pod port was changed to 8000, but referenced the service by name, allowing users to scrape the service at port 8080 for metrics. `v0.29.0` aligns the two ports so that service and pod metrics ports are the same. These ports are both set by `controller.metrics.port` and `webhook.port`, so if you have previously set these to non-default values, you may need to update your Prometheus scraper to match these new values.
```